playbooks/kafka_broker.yml

```diff
@@ -31,6 +31,17 @@
     - import_role:
         name: kafka_broker
 
+- name: Kafka Broker Serial Provisioning
```

Comment:
Can you add a comment describing this logic here? Is the idea that when kraft is enabled there is no way of querying the active controller anymore? is it possible we come up w new tasks that accomplish what dynamic_groups.yml does?